Signal/src/ViewControllers/ShowGroupMembersViewController.m

```diff
@@ -62,11 +100,15 @@ - (void)viewDidLoad {
 
     self.groupContacts =
         [[GroupContactsResult alloc] initWithMembersId:self.thread.groupModel.groupMemberIds without:nil];
-
+    __weak ShowGroupMembersViewController *weakSelf = self;
```

Comment:
I think I've fixed a serious memory leak that would prevent this view from every being deallocated.